app/controllers/games_controller.rb

```diff
@@ -9,7 +9,13 @@ def update
     @game.update_attributes(game_params)
     redirect_to_root_path
   end
-
+  
+  def destroy
+  end
+    @game = Game.find(params[:id])
```

Comment:
The logic should go between def / end ?

Reply:
haha yes...i don't know what happened there. i updated

Reply:
;-D even though I was pretty sure of this it feels weird to review other's code!